tests/sqllogictests/suites/mode/standalone/explain/window.test

```diff
@@ -285,5 +285,63 @@ Filter
         ├── push downs: [filters: [], limit: NONE]
         └── estimated rows: 0.00
 
+# test push down limit to window function
+statement ok
+drop table if exists t
+
+statement ok
+create table t(a int)
+
+statement ok
+insert into t values(1), (2), (3), (3)
+
+# range frame
+query T
+explain pipeline select a, sum(a) over (partition by a order by a desc) from t limit 3
+----
+CompoundBlockOperator(Project) × 1 processor
```

Comment:
I don't know if it's pushed or not from this pipeline.

Reply:
Yep, for range frame we cannot push limit to window sorts, the following rows frame will change `TransformSortMerge` => `TransformSortMergeLimit`

Reply:
I'll add the ranking functions cause they are frame insensitive.

Reply:
why not use explain select but use explain pipeline?

Reply:
the explain are same，we just reduce the sort with limit for window input，but still need sort